### ISSUE TITLE
maks antall tegn på bekreftcheckbokspanel bør reduseres til 80 tegn

### ISSUE THREAD

**Issue by karinaldw** [navikt/Designsystemet-old#164]
Antall tegn pr linje: 90
WCAG anbefaler 80 tegn. Jf. 1.4.8 Visuell presentasjon AAA 
https://uu.difi.no/krav-og-regelverk/wcag-20-standarden/ikke-lovpalagte-krav/148-visuell-presentasjon-niva-aaa

**Comment by randisolli** [comment]
Diskusjon på Slack
https://nav-it.slack.com/archives/G7NE6ESLX/p1585216558002800

**Comment by randisolli** [comment]
@KenAJoh , sjekk hvor vanskelig denne er å fikse. Vi ser først på bekreftcheckbokspanel, og kan senere se på generell brødtekst.
